jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java

```diff
@@ -238,7 +238,8 @@ public void storyCancelled(Story story, StoryDuration storyDuration) {
 
     @Override
     public void beforeStory(Story story, boolean givenStory) {
-        print(format("beforeStory", "{0}\n({1})\n", story.getDescription().asString(), story.getPath()));
+        print(format("beforeStory", "{2}\n{0}\n({1})\n", story.getDescription().asString(), story.getPath(),
```

Comment:
why not to have `"{0}\n{1}\n({2})\n"` and adjust the sequence of arguments?

Reply:
the same for other similar places

Reply:
it's separate task